src/backend/auth/adapter.ts

```diff
@@ -0,0 +1,195 @@
+import { Adapter, AdapterSession, AdapterUser } from "next-auth/adapters";
```

Comment:
OOC, why did you choose not to use the existing adapters, for say Prisma or Sequelize?

Also, add a link to the interface being implemented in the comments in this file: https://next-auth.js.org/tutorials/creating-a-database-adapter

Reply:
please see new comment further down in this file